lib/ain-ocean/src/model/mod.rs

```diff
@@ -21,7 +22,9 @@ mod transaction_vin;
 mod transaction_vout;
 mod tx_result;
 mod vault_auction_batch_history;
+// use ain_dftx::governance;
```

Comment:
```suggestion
```

Reply:
I cannot use above `governance `, due to amount 1.23 to 1.230000 conversion and some params are skipped as mentioned above, I need to serialisation.

Reply:
hmm.. i don't get it.. i think i just asked for removing unuse 🤔